Kickstarter-iOS/Assets.xcassets/icons/arrow-down-large.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
could we add the sketch files for these individual icons too? i've found it very useful to be able to have those files for re-exporting espesh when they needed to be pixel fitted